File: exercicios/cpp/Instruções Sequenciais/Q20.cpp
/*
------------------------------------------------
20. Crie um algoritmo/fluxograma/programa que 
leia a altura, a base maior e a base menor 
referentes a um trapézio, calcule e mostre na 
tela a sua área.
 */
#include <iostream>
#include <cmath>
#include <clocale>

using namespace std;
int main () {
	setlocale(LC_ALL,"Portuguese");
	double area, b_maior, b_menor, alt;
	cout<<"Informe a base maior:";
	cin>>b_maior;
	cout<<"Informe a base menor:";
	cin>>b_menor;    
	cout<<"Informe a altura:";
	cin>>alt;  
    area = ((b_maior + b_menor)*alt)/2;    
    cout<<"A área do trapézio é:"<<area<<endl;
	return 0;
}

File: exercicios/python/Instruções Sequenciais/Q20.py
# Exercício 20 — Área de trapézio
def main():
    b_maior = float(input("Base maior: "))
    b_menor = float(input("Base menor: "))
    altura = float(input("Altura: "))
    area = (b_maior + b_menor) * altura / 2
    print(f"Área do trapézio: {area:.2f}")

if __name__ == "__main__":
    main()